src/applications/mhv-medications/tests/e2e/utils/constants.js

```diff
@@ -92,6 +92,8 @@ export const Data = {
   DOWNLOAD_SUCCESS_CONFIRMATION_MESSAGE: 'Download started',
   DOWNLOAD_SUCCESS_ALERT_CONTENT:
     'Your file should download automatically. If it doesn’t, try again. If you can’t find it, check your browser settings to find where your browser saves downloaded files.',
+  REFILL_REQUEST_ERROR_ALERT_TEXT:
+    'Request not submittedWe’re sorry. There’s a problem with our system.Try requesting your refills again. If it still doesn’t work, contact your VA pharmacy.',
```

Comment:
looks like you need a space on this part `submittedWe’re` and this part `system.Try `